app/views/layouts/application.html.erb

```diff
@@ -44,6 +44,13 @@
       <% end %>
 
       <%= tag.div class: class_names("mx-auto w-full h-full", Current.user.show_sidebar? ? "max-w-4xl" : "max-w-5xl"), data: { sidebar_target: "content" } do %>
+        <% unless controller_path.start_with?('settings/') %>
+          <% if content_for?(:page_header) %>
+            <%= yield :page_header %>
+          <% else %>
+            <%= render "layouts/shared/page_header", breadcrumbs: current_breadcrumbs %>
+          <% end %>
+        <% end %>
```

Comment:
Since each of our page headers have varying elements in them (not just a title + subtitle), I would prefer keeping this solely to encapsulate the breadcrumbs UI logic and let the pages themselves handle the header content.  So in the layout:

```erb
<% if content_for?(:breadcrumbs) %>
  <%= yield :breadcrumbs %>
<% else %>
  <%= render "layouts/shared/breadcrumbs", breadcrumbs: @breadcrumbs %>
<% end %>
```

Where `shared/breadcrumbs` is:

```erb
<%# locals: (breadcrumbs:) %> 

<div class="mb-6">
  ... crumbs
</div>
```